index.js

```diff
@@ -1,17 +1,17 @@
 'use strict'
 
-var Raven = require('raven')
+const raven = require('raven')
 
-exports.register = function (server, options, next) {
-  Raven.config(options.dsn, options.client)
-  server.expose('client', Raven)
-  server.on('request-error', function (request, err) {
-    var baseUrl = request.info.uri ||
-      request.info.host && `${server.info.protocol}://${request.info.host}` ||
+exports.register = (server, options) => {
+  raven.config(options.dsn, options.client)
+  server.expose('client', raven)
+  server.events.on({ name: 'request', channels: 'error' }, (request, { error }) => {
```

Comment:
no spaces around object literals

Reply:
use `request, event` and don't destructure

Reply:
and destructuring

Reply:
That's fine, I don't personally want them. First is just style, destructuring is less clear here.